README.md

```diff
@@ -56,7 +56,6 @@ Parameters have default values set in `manifests/params.pp`.
 * Ubuntu 12.04 - Precise
 * Ubuntu 14.04 - Trusty
 * Ubuntu 16.04 - Xenial
-* Ubuntu 17.10 - Artful
```

Comment:
You removed this here but its also exists in the metadata.json 

https://github.com/logdna/puppet-logdna/blob/master/metadata.json#L22

Why are we removing Artful but not Precise (since its 7 years old) ?

Reply:
@gregswift, nice catch! because of this: https://fridge.ubuntu.com/2018/07/19/ubuntu-17-10-artful-aardvark-end-of-life-reached-on-july-19-2018/

Reply:
@gregswift, updated + gonna add higher versions of Ubuntu support in the upcoming updates!